lib/src/main/java/io/github/taz03/jia/InstagramClient.java

```diff
@@ -124,6 +163,10 @@ public LoginResponse login(Supplier<String> verificationCodeSupplier) throws Exc
         }
 
         if (loginResponse.getStatus().equals("ok")) this.pk = loginResponse.getUser().getProfile().getPk();
+
+        loginResponse.setAndroidId(androidId);
+        loginResponse.setAuthorization(authorization);
+        loginResponse.setUseragent(userAgent);
```

Comment:
isn't that already set

Reply:
How ?

Reply:
u set those in constructor calls, and send it in login method